Fase-1/API/self-order-management/schema.sql

```diff
@@ -1,27 +1,30 @@
-CREATE TABLE IF NOT EXISTS item
+CREATE TABLE IF NOT EXISTS product
 (
-    item_name VARCHAR(255) NOT NULL PRIMARY KEY,
-    item_type VARCHAR(255) NOT NULL,
-    item_cost_price NUMERIC(15,2) NOT NULL,
-    item_description VARCHAR(255),
-    item_category VARCHAR(255) NOT NULL
+    product_number INTEGER NOT NULL PRIMARY KEY,
+    product_name VARCHAR(255) NOT NULL,
+    product_type VARCHAR(255) NOT NULL,
+    product_cost_price NUMERIC(15,2) NOT NULL,
+    product_description VARCHAR(255),
+    product_min_sub_item INTEGER,
+    product_max_sub_item INTEGER,
+    product_category VARCHAR(255) NOT NULL
 );
 
 CREATE TABLE IF NOT EXISTS stock
 (
-    stock_item_name VARCHAR(255) PRIMARY KEY,
+    stock_product_number INTEGER NOT NULL PRIMARY KEY,
```

Comment:
Uma observação: nosso estoque é simples, com um relacionamento one-to-one entre produto e item no estoque. Hambúrgueres são compostos por vários ingredientes, como o hambúrguer em si, pão, queijo, etc. Eu gosto da nossa abordagem mais simples: cada produto com uma quantidade. Nesse caso, a unidade de medida seria sempre unitária, e o atributo `unit` nessa tabela parece inútil.

Btw, gosto da nossa abordagem mais simples.